Tidy: removes an empty css file and six unreferenced images from the css directory

vingle opened the pull request:
Overview
----------------------------------------
These images look like they are from a very old version of Civi, while abtesting.css is an empty file and I can't find any reference to it.

Before
----------------------------------------
Three image files and an empty css file in the css directory

After
----------------------------------------
Gone.

Technical Details
----------------------------------------
I've checked these four files aren't referenced anywhere with a Github UI search; but of course they may be referenced by some other means. But the images I've not seen and the worst case in not loading an empty css file is giving a file not found somewhere.

vingle committed:
Removes empty file

vingle committed:
Delete css/bg_button_a.gif

vingle committed:
Delete css/bg_button_span.gif

vingle committed:
Delete css/arrow.png

I can't find any use cases of this.

vingle commented:
I also can't find reference to [bluemarine.css](https://github.com/civicrm/civicrm-core/blob/master/css/bluemarine.css), so don't know when or if it's called, but it has too many styles in there for me to chance it. Maybe someone else knows?

colemanw commented:
> I also can't find reference to [bluemarine.css](https://github.com/civicrm/civicrm-core/blob/master/css/bluemarine.css), so don't know when or if it's called, but it has too many styles in there for me to chance it. Maybe someone else knows?

Sounds like a Drupal 5 thing.

vingle committed:
Delete css/bluemarine.css

Relates to an old (v4/5/6) Drupal theme with < 200 users (https://www.drupal.org/project/bluemarine)

vingle committed:
Merge branch 'civicrm:master' into CSS-folder-tidy

vingle committed:
Delete css/menu-collapsed.png

Can't see this called anywhere anymore.

vingle committed:
Delete css/menu-expanded.png

Can't see this used any more.

vingle commented:
Oh yup https://www.drupal.org/project/bluemarine. 

I've removed that and also two menu expand/collapse icons which I don't think are being used either (were added 12 years ago).
